src/styles/index.ts

```diff
@@ -4690,6 +4706,19 @@ const styles = (theme: ThemeColors) =>
             ...headlineFont,
             fontSize: variables.fontSizeXLarge,
         },
+
+        tripReservationIconContainer: {
+            width: 32,
+            height: 32,
+            backgroundColor: theme.overlay,
+            borderRadius: 32,
+            alignItems: 'center',
+            justifyContent: 'center',
+        },
+
+        tripDescriptionMargin: {
+            marginBottom: 2,
+        },
```

Comment:
Is this value valid? We usually have multiples of 4. 
NAB: Do we need to create a new style with only one value? Shouldn't we just use `styles.mb1` or something like this?

Reply:
![Screenshot 2024-03-29 at 12 29 23](https://github.com/Expensify/App/assets/47774969/6a11a1a3-9a64-4a57-9da9-fee5228a7c11)

Following this design, I've created a new style where the gap value is 2

Reply:
I think I would be fine using a gap of 4px since that technically adheres to the system better. And then this way we don't need to write a super specific style.